examples/tabular/example_custom_model_tabular.py

```diff
@@ -14,10 +14,10 @@
 
 # In this example, we create a custom Naive Bayes model for use in AutoGluon
 class NaiveBayesModel(AbstractModel):
-    def preprocess(self, X):
+    def _preprocess(self, X, **kwargs):
         cat_columns = X.select_dtypes(['category', 'object']).columns
         X = X.drop(cat_columns, axis=1)
-        return super().preprocess(X).fillna(0)
+        return super()._preprocess(X, **kwargs).fillna(0)
```

Comment:
could be helpful to also have toy example of _process_stateful in this model as well

Reply:
Agreed, I plan to overhaul this example into a notebook tutorial for v0.1, with more advanced model options being demonstrated.